jsonnet/kube-prometheus/components/node-exporter.libsonnet

```diff
@@ -178,8 +178,8 @@ function(params) {
         // NOTE: ignore veth network interface associated with containers.
         // OVN renames veth.* to <rand-hex>@if<X> where X is /sys/class/net/<if>/ifindex
         // thus [a-z0-9] regex below
-        '--collector.netclass.ignored-devices=^(veth.*|[a-z0-9]+@if\\d+)$',
-        '--collector.netdev.device-exclude=^(veth.*|[a-z0-9]+@if\\d+)$',
+        '--collector.netclass.ignored-devices=^(veth.*|[a-f0-9]{15})$',
```

Comment:
Is there any other, more generic way of writing this? 

Relevant discussion in https://github.com/prometheus-operator/kube-prometheus/pull/1224#discussion_r661252578

Reply:
The node exporter reads from devices from the /sys/class/net
https://github.com/prometheus/node_exporter#enabled-by-default
 and this only has 15 characters when using OVN.

Reply:
Yes, correct. My question is about making this regex OVN-agnostic. I.e. can we make this applicable to other CNI implementations? 
If there is no generic way, let's go with what is in this PR.

Reply:
My understanding is that veth.* will catch it all other CNI implementations. It is only in the case of OVN that it is being renamed.